code/addons/controls/package.json

```diff
@@ -51,24 +51,19 @@
     "prep": "node --loader ../../../scripts/node_modules/esbuild-register/loader.js -r ../../../scripts/node_modules/esbuild-register/register.js ../../../scripts/prepare/addon-bundle.ts"
   },
   "dependencies": {
-    "@storybook/blocks": "workspace:*",
     "dequal": "^2.0.2",
     "lodash": "^4.17.21",
     "ts-dedent": "^2.0.0"
   },
   "devDependencies": {
-    "@storybook/client-logger": "workspace:*",
-    "@storybook/components": "workspace:*",
-    "@storybook/core-common": "workspace:*",
+    "@storybook/blocks": "workspace:*",
```

Comment:
Why did this dependency change from direct to dev dependency (I'm assuming it will now be prebundled)? Maybe this explains the change in storybook-static size for the addon-controls files?

Reply:
Yes, because `@storybook/blocks` has a react peerDep, and I wanted to not have that anymore.

I guess this is where the 700kb increase is coming from.

We should move the ArgsTable component to `@storybook/core/components`!